helm/crsync/values.yaml

```diff
@@ -1,5 +1,7 @@
 lastModified: 1h
 
+syncPrivateRepositories: false
```

Comment:
Would it make sense to add a `flags` key and have `flags.lastModified`, `flags.syncPrivateRepositories` etc. WDYT?